Simulation/Pathfinding/ActionPathAux.cpp

```diff
@@ -14,17 +14,24 @@ ActionPathAux::ActionPathAux(const ActionPathAux &a){
 
 Path ActionPathAux::getPath() const{
     Path path{};
+
     path.actions.emplace(path.actions.begin(), this->action);
     path.cost = this->action.duration;
+    
     std::shared_ptr<ActionPathAux> predecessor = this->predecessor;
     while (predecessor != nullptr){
         path.actions.emplace(path.actions.begin(), predecessor->action);
         path.cost += predecessor->action.duration;
         predecessor = predecessor->predecessor;
     }
-    // Add a wait action at the end which is the work time
+    
+    // Adding timestamp, so the cost is the complete cost it took to reach this from start
+    path.cost += path.actions.front().timestamp;
```

Comment:
Hvorfor er det her nødvendigt?

Reply:
Kan måske også gøres anderledes, men gør det rigtig nemt at regne sum of all costs etc ud + lave udregninger i experimenter. Men man kan nok også bare lave noget path.actions.front() + path.cost i de beregninger det drejer sig om